tests/loaders_test.py

```diff
@@ -15,7 +15,9 @@
 from elastalert.loaders import RulesLoader
 from elastalert.util import EAException
 
-test_config = {'rules_folder': './empty_folder_test',
+empty_folder_test_path = os.path.join(os.path.dirname(__file__), './empty_folder_test')
```

Comment:
Since you're concatenating the paths together is the `./` still necessary?

Reply:
Removed.  It wasn't required before either actually.